### PR TITLE
Change CSV template

### PR DESCRIPTION
**Pull request by carlosthe19916**
https://issues.redhat.com/browse/TACKLE-312
Depends on https://github.com/konveyor/tackle-application-inventory/pull/136

This PR adds 2 new columns to the CSV template file available in the UI.

To download the CSV template click on `Download CSV template`:

Screenshot from 2021-09-21 10-20-23

### CODE REVIEW

File: public/sample_application_import.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Record Type 1,Application Name,Description,Comments,Business Service,Tag Type 1,Tag 1,Tag Type 2,Tag 2,Tag Type 3,Tag 3,Tag Type 4,Tag 4,Tag Type 5,Tag 5,Tag Type 6,Tag 6,Tag Type 7,Tag 7,Tag Type 8,Tag 8,Tag Type 9,Tag 9,Tag Type 10,Tag 10,Tag Type 11,Tag 11,Tag Type 12,Tag 12,Tag Type 13,Tag 13,Tag Type 14,Tag 14,Tag Type 15,Tag 15,Tag Type 16,Tag 16,Tag Type 17,Tag 17,Tag Type 18,Tag 18,Tag Type 19,Tag 19,Tag Type 20,Tag 20
-1,My application name,My application description,My comments,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Record Type 1,Application Name,Description,Comments,Business Service,Tag Type 1,Tag 1,Tag Type 2,Tag 2,Tag Type 3,Tag 3,Tag Type 4,Tag 4,Tag Type 5,Tag 5,Tag Type 6,Tag 6,Tag Type 7,Tag 7,Tag Type 8,Tag 8,Tag Type 9,Tag 9,Tag Type 10,Tag 10,Tag Type 11,Tag 11,Tag Type 12,Tag 12,Tag Type 13,Tag 13,Tag Type 14,Tag 14,Tag Type 15,Tag 15,Tag Type 16,Tag 16,Tag Type 17,Tag 17,Tag Type 18,Tag 18,Tag Type 19,Tag 19,Tag Type 20,Tag 20,Dependency,Dependency Direction
```
Comment (m-brophy) [inline public/sample_application_import.csv:1]: Phil was talking about having the two new fields towards the left in the file (before all the tags anyway) just so if you're creating a type 2 file you don't have to scroll all the way to the right to find the appropriate columns

File: public/sample_application_import.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+1,My application name,My application description,My comments,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (m-brophy) [inline public/sample_application_import.csv:2]: Maybe we could change the data row's `Record Type 1` field so it's not pre-populated with 1 ( 'Either 1 or 2'?). Is it worth also populating the Dependency Direction field with thw acceptable validated values ('Northbound or Southbound')?

File: public/sample_application_import.csv (hunk 1 above)
Comment (carlosthe19916) [reply public/sample_application_import.csv]: `Dependency,Dependency Direction` are now between `Business service` and `Tag types`. What do you think about it?

File: public/sample_application_import.csv (hunk 2 above)
Comment (carlosthe19916) [reply public/sample_application_import.csv]: I'm not sure if this is a good idea but I added 2 new rows to the CSV so the first 2 rows create applications and the 3rd row creates a dependency between the previously created apps. What do you think about it?

**Pull request merged**